cli/Tests/TuistGeneratorTests/Generator/SwiftPackageManagerInteractorTests.swift

```diff
@@ -223,55 +222,6 @@ final class SwiftPackageManagerInteractorTests: TuistTestCase {
         XCTAssertFalse(exists)
     }
 
-    func test_generate_sets_cloned_source_packages_dir_path() async throws {
```

Comment:
I'd not delete this test, since it's still possible to pass `clonedSourcePackagesDirPath` and we don't want to break that functionality. Same [here](https://github.com/tuist/tuist/pull/8099/files#diff-4d696917842982c6e64335cb5b35218e6278e260742a7d08a5a3e6ff633f066eR102). I'd keep the test, and add a new one for the scenario where you pass the arguments manually.